include/bluetooth/services/ancs_c.h

```diff
@@ -0,0 +1,580 @@
+ * arrive in the @ref BT_GATT_ANCS_C_EVT_NOTIF_ATTRIBUTE event.
+ * Use @ref bt_gatt_ancs_c_app_attr_request to request attributes of the app that issued
+ * the notifications. The app attributes arrive in the @ref BT_GATT_ANCS_C_EVT_APP_ATTRIBUTE event.
+ * Use @ref bt_gatt_ancs_perform_notif_action to make the Notification Provider perform an
+ * action based on the provided notification.
+ *
+ * @msc
+ * hscale = "1.5";
+ * Application, ANCS_C;
+ * |||;
+ * Application=>ANCS_C   [label = "bt_gatt_ancs_c_attr_add(attribute)"];
+ * Application=>ANCS_C   [label = "bt_gatt_ancs_c_init(ancs_instance, event_handler)"];
+ * ...;
+ * Application=>ANCS_C   [label = "bt_gatt_ancs_c_handles_assign(ancs_instance, conn_handle, service_handles)"];
+ * Application=>ANCS_C   [label = "bt_gatt_ancs_c_notif_source_notif_enable(ancs_instance)"];
+ * Application=>ANCS_C   [label = "bt_gatt_ancs_c_data_source_notif_enable(ancs_instance)"];
+ * |||;
+ * ...;
+ * |||;
+ * Application<<=ANCS_C  [label = "BT_GATT_ANCS_C_EVT_NOTIF"];
+ * |||;
+ * ...;
+ * |||;
+ * Application=>ANCS_C   [label = "bt_gatt_ancs_c_request_attrs(attr_id, buffer)"];
+ * Application<<=ANCS_C  [label = "BT_GATT_ANCS_C_EVT_NOTIF_ATTRIBUTE"];
+ * |||;
+ * @endmsc
+ */
+
+#include <bluetooth/gatt.h>
+#include <bluetooth/conn.h>
+#include <bluetooth/gatt_dm.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** @brief Apple Notification Center Service UUID. */
+#define ANCS_UUID_SERVICE                                                      \
+	BT_UUID_128_ENCODE(0x7905f431, 0xb5ce, 0x4e99, 0xa40f, 0x4b1e122d00d0)
+
+/** @brief Notification Source Characteristic UUID. */
+#define ANCS_UUID_CHAR_NOTIFICATION_SOURCE                                     \
+	BT_UUID_128_ENCODE(0x9fbf120d, 0x6301, 0x42d9, 0x8c58, 0x25e699a21dbd)
+
+/** @brief Control Point Characteristic UUID. */
+#define ANCS_UUID_CHAR_CONTROL_POINT                                           \
+	BT_UUID_128_ENCODE(0x69d1d8f3, 0x45e1, 0x49a8, 0x9821, 0x9bbdfdaad9d9)
+
+/** @brief Data Source Characteristic UUID. */
+#define ANCS_UUID_CHAR_DATA_SOURCE                                             \
+	BT_UUID_128_ENCODE(0x22eac6e9, 0x24d6, 0x4bb5, 0xbe44, 0xb36ace7c7bfb)
+
+#define BT_UUID_ANCS_SERVICE BT_UUID_DECLARE_128(ANCS_UUID_SERVICE)
+#define BT_UUID_ANCS_NOTIFICATION_SOURCE                                       \
+	BT_UUID_DECLARE_128(ANCS_UUID_CHAR_NOTIFICATION_SOURCE)
+#define BT_UUID_ANCS_CONTROL_POINT                                             \
+	BT_UUID_DECLARE_128(ANCS_UUID_CHAR_CONTROL_POINT)
+#define BT_UUID_ANCS_DATA_SOURCE BT_UUID_DECLARE_128(ANCS_UUID_CHAR_DATA_SOURCE)
+
+/** Maximum data length of an iOS notification attribute. */
+#define BT_GATT_ANCS_ATTR_DATA_MAX 32
+/** Number of iOS notification categories: Other, Incoming Call, Missed Call, Voice Mail, Social, Schedule, Email, News, Health and Fitness, Business and Finance, Location, Entertainment. */
```

Comment:
Lines should stay within 80 characters.

Reply:
Fixed.
Compliance check mandates 100 characters. Other places are modified to get through the check.